geopandas/tests/test_array.py

```diff
@@ -158,6 +158,16 @@ def test_from_wkb():
     assert res[0] == multi_poly
 
 
+def test_from_wkb__hex():
```

Comment:
```suggestion
def test_from_wkb_hex():
```

I think that this is the only suggestion I can make here. We can almost merge :).